torch/csrc/Module.cpp

```diff
@@ -908,6 +908,32 @@ PyObject* initModule() {
   py_module.def("_demangle", &c10::demangle);
   py_module.def("_log_api_usage_once", &LogAPIUsageOnceFromPython);
 
+  py::enum_<c10::LogLevel>(py_module, "LogLevel")
+        .value("FATAL",   c10::LogLevel::Fatal)
+        .value("ERROR",   c10::LogLevel::Error)
+        .value("WARNING", c10::LogLevel::Warning)
+        .value("INFO",    c10::LogLevel::Info);
+
+  py_module.def(
+    "set_c10_log_level",
```

Comment:
I don't think we ever expose `c10` as name on Python level, why do it here?
```suggestion
    "set_log_level",
```

Reply:
My intention was to avoid confusion between Python level logging versus C++ level logging. This function only changes the log level of the C++ logger. I don't have a strong opinion on this, but I still think that being a bit more specific in its name might help users. As an end-user when I call a function named `torch.set_log_level()` I would expect it to have a global effect. However since many of our Python APIs use the `logging` module as well which don't respect this setting, it might get confusing.